src/atomate2/vasp/jobs/mp.py

```diff
@@ -0,0 +1,217 @@
+"""
+Module defining jobs for Materials Project r2SCAN workflows.
+
+Reference: https://doi.org/10.1103/PhysRevMaterials.6.013801
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.sets.mp import MPGGARelaxGenerator, MPMetaGGARelaxGenerator
+
+if TYPE_CHECKING:
+    from atomate2.vasp.sets.base import VaspInputGenerator
+
+__all__ = ["MPPreRelaxMaker", "MPMetaGGARelaxMaker", "MPMetaGGAStaticMaker"]
+
+
+@dataclass
+class MPGGARelaxMaker(BaseVaspMaker):
+    """
+    Maker to create VASP relaxation job using PBE GGA by default.
+
+    Parameters
+    ----------
+    name : str
+        The job name.
+    input_set_generator : .VaspInputGenerator
+        A generator used to make the input set.
+    write_input_set_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.write_vasp_input_set`.
+    copy_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.copy_vasp_outputs`.
+    run_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.run_vasp`.
+    task_document_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.TaskDoc.from_directory`.
+    stop_children_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.should_stop_children`.
+    write_additional_data : dict
+        Additional data to write to the current directory. Given as a dict of
+        {filename: data}. Note that if using FireWorks, dictionary keys cannot contain
+        the "." character which is typically used to denote file extensions. To avoid
+        this, use the ":" character, which will automatically be converted to ".". E.g.
+        ``{"my_file:txt": "contents of the file"}``.
+    """
+
+    name: str = "MP GGA Relax"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: MPGGARelaxGenerator(auto_ismear=False)
```

Comment:
If the default value of `auto_ismear` for this calculation is False, then you can set that directly in the `MPGGARelaxGenerator` rather than here.